MAVProxy/mavproxy.py

```diff
@@ -847,9 +848,13 @@ def input_loop():
         except EOFError:
             mpstate.status.exit = True
             sys.exit(1)
-        mpstate.input_queue.put(line)
-
-
+        mpstate.input_count += 1
+        cmds = line.split(';')
+        if len(cmds) == 1 and cmds[0] == "":
+              mpstate.empty_input_count += 1
+        for c in cmds:
+            process_stdin(c)
```

Comment:
calling `module load map` here works, but the same command here https://github.com/SamuelDudley/MAVProxy/blob/357239cb97fd6b50d1fc8087b24502bfc75b67fa/MAVProxy/mavproxy.py#L771 fails silently. 
It only seems to affect multiprocessing, child threads can run fine.